norfair/drawing/utils.py

```diff
@@ -13,12 +13,16 @@ def _centroid(tracked_points: np.ndarray) -> Tuple[int, int]:
     return int(sum_x / num_points), int(sum_y / num_points)
 
 
-def _build_text(drawable: "Drawable", draw_labels, draw_ids):
+def _build_text(drawable: "Drawable", draw_labels, draw_ids, draw_scores):
     text = ""
     if draw_labels and drawable.label is not None:
         text = str(drawable.label)
     if draw_ids and drawable.id is not None:
         if len(text) > 0:
             text += "-"
         text += str(drawable.id)
+    if draw_scores and drawable.scores is not None:
+        if len(text) > 0:
+            text += "-"
+        text += str(np.round(drawable.scores[0],4))
```

Comment:
For bounding boxes, all scores are the same but for keypoints this might not be the case, for this reason, I think is better to draw the average of all scores instead of the first one